common/lib/kontena-logging.rb

```diff
@@ -0,0 +1,3 @@
+module Kontena
+  require_relative './kontena/logging'
```

Comment:
This is there to define the `Kontena` module... just `require 'kontena/logging'` will fail if it's the first thing being required, and nothing else has yet done `module Kontena`.

Could change these to just use `require 'kontena/logging'`, but then each class would have to be like

```ruby
module Kontena
  module Foo
    class Bar
      # actual code with too much indentation
```

Not so much a problem for the basic logging thing, but annoying enough already for the `Kontena::Websocket::Client`.